### PR TITLE
Download gb by taxid

### PR DESCRIPTION
**Pull request by bosborne**
### New Features

Add capability for Genbank downloads

### CODE REVIEW

**Comment by bosborne** [comment]
Not clear to me how to associate the files with the DD though.

File: covid19-etl/etl/download_gb_by_taxid.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+taxid: 694009
+email: briano@bioteam.net
```
Comment (paulineribeyre) [inline covid19-etl/etl/download_gb_by_taxid.yaml:3]: this repo is public, are you comfortable sharing this email address? you could leave it blank and add a comment instead - can this be any email address, or it is the email of someone who has access to controlled access data?

Comment (bosborne) [reply covid19-etl/etl/download_gb_by_taxid.yaml:3]: @paulineribeyre There's a more fundamental issue, this PR no longer has the latest/best code. I'd rather we just not do the merge, give me a day or 2 and I'll get you more complete code. So, suggest you not accept this PR.

Comment (m0nhawk) [reply covid19-etl/etl/download_gb_by_taxid.yaml:3]: I saw another PR from the fork repository, but now it looks like removed (the fork), so I closed that PR. But it has the same changes. https://github.com/uc-cdis/covid19-tools/pull/12

**Comment by bosborne** [comment]
Right, let’s reject both (all) of these PRs, my understanding of what the code is supposed to do is better now.

**Pull request closed without merging**